.env.e2e

```
@@ -0,0 +1,93 @@
+
+# Test Data Configuration
+TEST_DATA_PATH=/app/test_data
+TEST_DATABASE_RESET=true
+
+# Logging Configuration
+LOG_LEVEL=info
+LOG_FORMAT=json
+
+# Feature Flags for Testing
+ENABLE_SIGNUP=true
+ENABLE_EMAIL_VERIFICATION=false
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The ENABLE_EMAIL_VERIFICATION key should go before the ENABLE_SIGNUP key

(UnorderedKey)